packages/x-date-pickers-pro/src/SingleInputDateRangeField/tests/editing.SingleInputDateRangeField.test.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { spy } from 'sinon';
+import { SingleInputDateRangeField } from '@mui/x-date-pickers-pro/SingleInputDateRangeField';
+import { screen, userEvent, fireEvent } from '@mui/monorepo/test/utils';
+import { expectInputValue } from 'test/utils/pickers-utils';
+import { describeAdapters } from '@mui/x-date-pickers/tests/describeAdapters';
+
+describe('<SingleInputDateRangeField /> - Editing', () => {
```

Comment:
I copied and adapted all the relevant from the `Backspace` `describe` of `editing.DateField.test.tsx`.
Not all are related to this PR